Blaise-Shyaka/My-Mentor
# [finishes #170363654] Complete view all mentors page

# What does this PR do?

This is a view all mentors page

# Description of tasks to be completed

- Build a responsive "view all mentors" page
- Button allowing a user to view all details about the mentor
- Button allowing a user to request a mentorship session

# How should this manually be tested?

- Switch to the views directory and open mentors.html page. Resize the browser to verify its responsiveness
- Click buttons to verify their redirection to other pages
